Libraries/Components/TextInput/TextInput.js

```diff
@@ -1601,6 +1602,14 @@ const ExportedForwardRef: React.AbstractComponent<
     React.ElementRef<HostComponent<mixed>> & ImperativeMethods,
   >,
 ) {
+  let style = flattenStyle(restProps.style);
+
+  if (style && style.verticalAlign !== undefined) {
```

Comment:
Lets use optional chaining here,

```suggestion
  if (style?.verticalAlign  != null) {
```

Reply:
Sure, updated